crates/solver/src/settlement_post_processing/optimize_score.rs

```diff
@@ -42,12 +47,12 @@ pub async fn compute_score(
         .calculate(&inputs, nmb_orders)
         .map(Score::Score);
 
-    tracing::trace!(
+    tracing::debug!(
         "solver {} objective value {}, score {:?}",
         solver,
         inputs.objective_value(),
         score
     );
```

Comment:
Same here like Martin said above, something like
```suggestion
    tracing::debug!(
        %solver, ?score, objective_value = %inputs.objective_value(),
        "computed score"
    );
```

Reply:
fixed